src/features/readers/readersSlice.ts

```diff
@@ -65,6 +72,35 @@ const readersSlice = createSlice({
         state.isReadersLoading = false
         state.isReadersError = true
       })
+      .addCase(getSingleReader.pending, (state) => {
+        state.isReadersError = false
+        state.isReadersLoading = true
+      })
+      .addCase(getSingleReader.fulfilled, (state, { payload }) => {
+        state.singleReader = payload
+        state.isReadersLoading = false
+      })
+      .addCase(getSingleReader.rejected, (state) => {
+        state.isReadersError = true
+        state.isReadersLoading = false
+      })
+      .addCase(editReader.fulfilled, (state, { payload }) => {
+        if (payload && state.singleReader) {
+          return {
+            ...state,
+            singleReader: { ...state.singleReader, ...payload },
+          }
+        }
+      })
+      .addCase(deleteReaderPhoto.fulfilled, (state, { payload }) => {
+        if (
+          payload &&
+          state.singleReader &&
+          state.singleReader._id === payload
+        ) {
+          state.singleReader.photo = null
```

Comment:
After request to delete photo endpoint you should download the latest data by fetching single reader info